apollo.config.js

```diff
@@ -0,0 +1,16 @@
+require('dotenv').config();
+
+// Configuration for the vscode-apollo extension and other Apollo tooling.
+module.exports = {
+  client: {
+    service: {
+      name: 'are.na',
+      url: 'https://api.are.na/graphql',
+      headers: {
+        'X-APP-TOKEN': process.env.GRAPHQL_XAPP_TOKEN,
+      },
+    },
+    includes: ['src/**/*.ts', 'src/v2/apollo/localState/clientSchema.graphql'],
```

Comment:
This pulls in the `clientSchema` so that we can generate types and get validation and so on. Note that I typed each property in here as a string for now just to get things working, but as code is converted to TypeScript these types will likely need to be updated to match how they're actually used in code.